src/content/guides/advanced-coredns-configuration/index.md

```diff
@@ -9,27 +9,105 @@ tags: ["tutorial"]
 
 # Advanced CoreDNS Configuration
 
-The [CoreDNS addon](https://github.com/coredns/coredns) running inside your cluster has additional configuration options and features that can be customized.
+Your Giant Swarm installation comes with a default configuration for the [CoreDNS addon](https://github.com/coredns/coredns)
 
-You can customize two of these configuration options on a per cluster basis through a ConfigMap inside your clusters. The ConfigMap is named `coredns-user-values` and is located in the `kube-system` namespace.
+You can override these defaults in a ConfigMap named `coredns-user-values`.
 
-__Note:__ This feature is only available in more recent cluster versions. To check if your cluster version supports customization through the ConfigMap, you can check if the above-mentioned ConfigMap is present.
+## Where is the user values ConfigMap?
+
+Given the cluster you are trying to configure has id: `123ab`
+
+**Release Version 9.0.1 and greater:**
+
+If your cluster is on release version `9.0.1` or greater then you will find the `coredns-user-values` ConfigMap on the Control Plane in the `123ab` namespace:
 
 ```nohighlight
-$ kubectl -n kube-system get cm coredns-user-values
+$ kubectl -n 123ab get cm coredns-user-values --context=control-plane
 NAME                                   DATA      AGE
 coredns-user-values                    0         11m
 ```
 
-On cluster creation the ConfigMap is empty and below-mentioned defaults will be applied to the final CoreDNS deployment. To customize any of the configuration options, you just need to add the respective line(s) in the data field of the user ConfigMap.
+**Release Version 9.0.0 and below:**
+
+If the cluster has a release version equal to `9.0.0` or lower, then you will find the `coredns-user-values` ConfigMap on the Tenant Cluster itself in the `kube-system` namespace:
+
+```nohighlight
+$ kubectl -n kube-system get cm coredns-user-values --context=tenant-cluster
+NAME                                   DATA      AGE
+coredns-user-values         0         11m
+```
+
+-----
+
+Upgrading from `9.0.0` to a higher release will automatically migrate these user values from the Tenant Cluster to the
+Control Plane for you. If you have any automation or existing workflows you should keep this location change in mind.
+
+------
 
-__Warning:__ Please do not edit any of the other CoreDNS related resources. Only the user ConfigMap is safe to edit.
+__Warning:__
+
+Please do not edit any other coredns related ConfigMaps.
+
+Only the user values ConfigMap is safe to edit.
+
+------
+
```

Comment:
Is this right? It looks like broken in the preview

Reply:
hmm, looks how i intended it to, which is to give it a line above and below, for extra visibility